app.js

```diff
@@ -1,8 +1,55 @@
 // import functions and grab DOM elements
+const hatImg0 = document.getElementById('hat-0');
+const hatImg1 = document.getElementById('hat-1');
+const hatImg2 = document.getElementById('hat-2');
 
+const button0 = document.getElementById('button-0');
+const button1 = document.getElementById('button-1');
+const button2 = document.getElementById('button-2');
+
+const winsEl = document.getElementById('wins');
+const totalEl = document.getElementById('total');
 // let state
+let wins = 0;
+let total = 0;
+
+//Create function first
+
+function handleGuess(crownId){
+    hatImg0.src = 'assets/goldkinghat.png';
+    hatImg1.src = 'assets/goldkinghat.png';
+    hatImg2.src = 'assets/goldkinghat.png';
+
+    total++;
+
+    const numberBetweenZeroAndThree = Math.floor(Math.random() * 3);
```

Comment:
the rubric asks for this line to be pulled out and used in a new function called `getRandomItem(arr)`. 
it takes in the array, `const array = [ 'hat1', 'hat2', 'hat3' ]` and returns a random item from the array.